guide/designing-products/common-user-flows.md

```diff
@@ -293,10 +293,14 @@ We ask users to securely back up their keys when they create new wallets, to res
    layout = "float-left"
 %}
 
-Like fiat currencies, securely storing money can be as simple as storing some coins in your pocket or highly complex with multiple banks' safety deposit boxes. For self-custodial wallets, all of this is in the users' hands, although wallet software ideally provides guidelines and support to more easily follow best practices. Continue to the Key Management section for introductions and deep-dives into different techniques.
+Like fiat currencies, securely storing money can be as simple as storing some coins in your pocket or highly complex with multiple banks’ safety deposit boxes. For self-custodial wallets, all of this is in the users’ hands, although wallet software ideally provides guidelines and support to more easily follow best practices. See also, the [Private key management](/guide/private-key-management/introduction/) section and [Protecting a wallet](guide/onboarding/protecting-a-wallet/).
```

Comment:
```suggestion
Like fiat currencies, securely storing wealth can be as simple as storing some coins in your pocket or highly complex with multiple banks’ safety deposit boxes. For self-custodial wallets, all of this is in the users’ hands, although wallet software ideally provides guidelines and support to more easily follow best practices. See also, the [Private key management](/guide/private-key-management/introduction/) section and [Protecting a wallet](guide/onboarding/protecting-a-wallet/).
```

Read more clearly to me, fiat currencies and money seem synonymous in my mind.

Reply:
`Wealth` sounds like it's only for rich people. How about `funds`?

Reply:
👍 

Wealth doesn't sound to me for rich people, but we've used `funds` accros the guide already so I also prefer the funds.

Reply:
Changed in 7951c29